en/docs/setup/installing-on-linux-or-os-x.md

```diff
@@ -1,9 +1,7 @@
 # Installing on Linux or OS X
 
-!!! tip
-    
-    **Before you begin**, [please see our compatibility
-    matrix](https://docs.wso2.com/display/compatibility/Tested+Operating+Systems)
+!!! tip "Before you begin"
+    See the [compatibility matrix](../../administer/tested-operating-systems)
```

Comment:
there is no file called "tested-operating-systems" in the administer folder.
please check